src/features/session_replay/shared/recorder.js

```diff
@@ -74,8 +83,39 @@ export class Recorder {
     }
   }
 
+  /**
+   * audit - Checks if the event node payload is missing certain attributes
+   * will forward on to the "store" method if nothing needs async fixing
+   * @param {*} event - An RRWEB event node
+   * @param {*} isCheckout - Flag indicating if the payload was triggered as a checkout
+   */
+  audit (event, isCheckout) {
+    /** only run the audit if inline_stylesheets is configured as on (default behavior) */
+    if (this.shouldInlineStylesheets) {
+      this.currentBufferTarget.inlinedAllStylesheets = false
```

Comment:
Why is this flag set to false here? if inline_stylesheet isn't opted for, it doesn't necessarily mean the events _does not_ only have inline css

Reply:
fixed in https://github.com/newrelic/newrelic-browser-agent/pull/859/commits/fe6172e7c7a94898afe418b8cf2e56776d0fa858